java/SingUpTests.java

```diff
@@ -0,0 +1,136 @@
+import io.github.bonigarcia.wdm.WebDriverManager;
```

Comment:
1. Please add @before/after methods to this class
2. Add implicity wait 
3. Remove thread sleeps